draft-ietf-mls-protocol.md

```diff
@@ -1586,9 +1579,6 @@ psk_secret     = psk_secret_[0] || ... || psk_secret_[n-1]
 The `index` field in `PSKLabel` corresponds to the index of the PSK in the `psk`
 array, while the `count` field contains the total number of PSKs.
 
-<!-- OPEN ISSUE: How to combine multiple PSKs such that the final PSK, is
-pseudorandom if at least one of the PSKs used is pseudorandom. -->
```

Comment:
I don't think the security of the algorithm above an be reduced to a "standard assumption". This would be what the nKDF approach by @chrisbrzuska was for. I believe we discussed this in the group as a general approach for the key schedule, but then decided against it. There is a (somewhat) standard assumption for doing this with two keys (which we need for key schedule security), but I don't think anyone has claimed this for `n` keys.